apps/cowswap-frontend/src/modules/wallet/pure/StatusIcon/index.tsx

```diff
@@ -28,9 +28,6 @@ export function StatusIcon({ connectionType }: StatusIconProps) {
     case ConnectionType.INJECTED:
       image = <Identicon />
       break
-    case ConnectionType.WALLET_CONNECT:
-      image = <img src={WalletConnectIcon} alt="WalletConnect" />
-      break
```

Comment:
Doesn't v2 have an icon?

Reply:
It didn't before, but I think you are right, it should have. Added